contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer.py

```diff
@@ -72,17 +72,18 @@ def _execute_buildozer_script(self, command):
       Buildozer.execute_binary(command, address.spec, binary=self._executable)
 
   @classmethod
-  def execute_binary(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c'):
+  def execute_binary(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c', supress_warnings=False):
     binary = binary if binary else BinaryUtil.Factory.create().select_binary('scripts/buildozer', version, 'buildozer')
 
-    Buildozer._execute_buildozer_command([binary, command, spec])
+    Buildozer._execute_buildozer_command([binary, command, spec], supress_warnings)
 
   @classmethod
-  def _execute_buildozer_command(cls, buildozer_command):
+  def _execute_buildozer_command(cls, buildozer_command, supress_warnings):
```

Comment:
nit: s/supress/suppress/

Reply:
Updated to `suppress_warnings` in peek.py and buildozer.py.